src/collectors/aerospike/aerospike.py

```diff
@@ -0,0 +1,187 @@
+# coding=utf-8
+
+"""
+Collect statistics from Aerospike
+
+#### Dependencies
+
+ * socket
+ * telnetlib
```

Comment:
Using a core module felt easier to manage from a dependencies perspective (rather than having to deploy the python and C client on every server).  

I don’t think the ‘info’ API in the python client library is actually any cleaner.  It appears to return string blobs that are about the same as we get back from the info port.

Keep in mind I’m not bypassing or re-implementing the client here; I’m explicitly using the plaintext port.

> On Jul 5, 2016, at 6:51 PM, Grant Ridder <notifications@github.com> wrote:
> 
> In src/collectors/aerospike/aerospike.py <https://github.com/python-diamond/Diamond/pull/500#discussion_r69651706>:
> 
> > @@ -0,0 +1,187 @@
> > +# coding=utf-8
> > +
> > +"""
> > +Collect statistics from Aerospike
> > +
> > +#### Dependencies
> > +
> > + * socket
> > + * telnetlib
> Any reason for using telnet as opposed to the aerospike python library?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/python-diamond/Diamond/pull/500/files/fd2c8e482858397ceaa83fdf2ea220c326b90ad7#r69651706>, or mute the thread <https://github.com/notifications/unsubscribe/AF6VIioPco3JpNdjPGldKWJBCdv3EQmnks5qSt_9gaJpZM4JFmK8>.
>